bedrock/firefox/tests/test_views.py

```diff
@@ -654,66 +654,66 @@ def test_get_context_data_one_star(self):
 @override_settings(DEV=False)
 @patch('bedrock.firefox.views.l10n_utils.render')
 class TestFirefoxFeatures(TestCase):
```

Comment:
I was going to leave them in as safeguards against future changes to the view, but since I'm going to make these just `page` directives, I don't see a reason to keep the tests.